product/plan.md

```diff
@@ -13,6 +13,8 @@ users. This enables everyone to get the most out of FlowForge. FlowForge.com
 should always run the default configuration if possible to ensure the best
 experience across the board.
 
+This does not mean that we don't allow users to change settings where appropriate, setting sensible defaults means that we don't require users to make selections or enter information where at all possible, we use placeholders or generate a default value for fields, we ensure that dropdowns have the sensible default as the already selected option. 
```

Comment:
Struggling to make sense of what you're written if I'm honest. It's all one sentence, and has an opening double negative which is throwing me.

Reply:
I've re-worded it slightly, the key point I'm trying to get across is that sensible defaults doesn't mean no options.